library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -138,6 +138,11 @@ public void onInfoWindowClick(Marker marker) {
     callback.onMapBoundsReady(builder.build());
   }
 
+  @Override
+  public void getLatLngScreenLocation(LatLng latLng, OnLatLngScreenLocationCallback callback) {
```

Comment:
can you rename this to `getScreenLocation(LatLng latLng, Callback)`? the `LatLng` is implied by the existence of the `LatLng` argument